project/ntut-xuan/README.rst

```diff
@@ -0,0 +1,163 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+We are trying to shard the polygon with the gird. 
+The grid contains lot of cell. 
+
+If the cell is in the polygon, we will mark it and calculate the mark the cell 
+to get the approximately size.
+
+
+
+Problem to solve
+-------------------------------------------------------------------------------
+This application support that calculate the size of complex polygon. 
+It can be a normal polygon to get the approx area, and it can composite 
+lot of polygon to make a complex polygon and get the approx area.
+
+
+The procedure that we solve the problem:
+
+- Input the point sets for polygons
+- Adjust the specific polygon is union/difference/clip the base polygon.
+- Trying to mark the cell in grid.
+- Calculate how much cell is marked and calculate the approximately area.
+
+
+Prospective Users
+-------------------------------------------------------------------------------
+This application can help the user that trying to get the size of 
+complex polygon. 
+
+Since it may complex to calculate the polygon composite lot of polygon, 
+it should be a nice tool to complete it.
+
+The user may trying to input points set file with CLI and adjust the bitwise 
+operator to describe the complex polygon. 
+
+The idea is based on how complex polygon described in SVG files.
+
+
+System Architecture
+-------------------------------------------------------------------------------
+In this software, we trying to implement numerical process in C/C++. 
+e would like to use pybind11 to convert C++ library into python module.
+
+We expect two classes `Polygon` and `Point` to describe the polygon and point. 
+
+Also, we may have a function to determine the cell is in the polygon or not by 
+describing the top-left coordinate of cell and the width/height of cell.
+
+
+API Description
+-------------------------------------------------------------------------------
+`Polygon` class will have a constructor and destructor 
+to input multiple `Point`. 
+The `Point` class will have a constructor to input the coordinate x and y. 
+
+**For more description we may need to design of it. 
+It will added in the future.**
+
+
+Engineering Infrastructure
+-------------------------------------------------------------------------------
+In this project, we have these infrastructure to make sure our project
+can develop properly.
+
+-  Version Control: Git to maintaince the version control and GitHub to
+   handle the repository.
+-  Build Tools (Package Tools): CMake for C/C++.
+-  Testing framework: GoogleTest for C/C++ and Pytest for Python. 
+   We will test all the utiltiy function or classes 
+   to make sure it keeps high test coverage.
+
+
+Schedule
+-------------------------------------------------------------------------------
+
+.. list-table::
+
+ * - Week
+   - Task 
+   - Progress (%)
+ * - 1
+   - System Design / Writing proposal
+   - (??/26)
```

Comment:
It is not clear what does `??/26` mean.  If you want to write down percentage, just write percentage.